webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -154,12 +154,47 @@ export default class CreateIssueModal extends PureComponent {
             return null;
         }
 
-        let component;
         if (error) {
             console.error('render error', error); //eslint-disable-line no-console
         }
 
-        if (!jiraIssueMetadata || !jiraIssueMetadata.projects) {
+        let component;
+        let footer = (
+            <React.Fragment>
+                <FormButton
+                    type='button'
+                    btnClass='btn-link'
+                    defaultMessage='Cancel'
+                    onClick={this.handleClose}
+                />
+                <FormButton
+                    type='submit'
+                    btnClass='btn btn-primary'
+                    saving={submitting}
+                >
+                    {'Create'}
+                </FormButton>
+            </React.Fragment>
+        );
+
+        if (jiraIssueMetadata && jiraIssueMetadata.error) {
+            component = (
+                <div style={style.modal}>
+                    {jiraIssueMetadata.error}
+                </div>
+            );
+
+            footer = (
+                <React.Fragment>
+                    <FormButton
+                        type='submit'
+                        btnClass='btn btn-primary'
+                        defaultMessage='Close'
+                        onClick={this.handleClose}
+                    />
+                </React.Fragment>
+            );
+        } else if (!jiraIssueMetadata || !jiraIssueMetadata.projects) {
```

Comment:
@jfrerich please note that the original PR also checked `post` here, but it's undefined in `master`. I am not sure if I missed something, or if it is correct that we no longer need it. Please confirm.

Reply:
I don't remember deleting checking for `post` and still have in my latest `jira2` branch.